### PR TITLE
Diverse improvements

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
Diverse tweaks to the project :)

### CODE REVIEW

**Comment by joshrlesch** [comment]
Awesome, thanks for the PR. I'll get to testing these soon!!

File: lib/fastlane/plugin/yarn/helper/yarn_helper.rb [hunk 1]
```diff
@@ -8,21 +8,27 @@ class YarnHelper
...
+      attr_accessor :project_root
       attr_accessor :yarn
 
-      def initialize(package_path: nil)
+      def initialize(package_path: nil, project_root:nil)
         self.package_path = package_path
 
         if self.package_path
           self.yarn = "cd #{File.dirname(self.package_path)} && yarn"
+        elsif
```
Comment (joshrlesch) [inline lib/fastlane/plugin/yarn/helper/yarn_helper.rb]: wont this `elsif` and `else` both be called if `package_path` is `nil`?

Comment (AdrieanKhisbe) [reply lib/fastlane/plugin/yarn/helper/yarn_helper.rb]: You are right, would have catch it if I dud not have forgotten to write a real test for project_root.
Pushed the fixed inn d7ff91b :)

**Comment by AdrieanKhisbe** [comment]
@joshrlesch Here is Gentle reminder :)

**Review by joshrlesch — APPROVED** [review]

**Comment by joshrlesch** [comment]
LGTM, ran them in my setup and everything still works. Thanks!

**Comment by AdrieanKhisbe** [comment]
Awesome :)

**Comment by AdrieanKhisbe** [comment]
Feel  free to ping me when released :)

**Comment by joshrlesch** [comment]
@AdrieanKhisbe released!

**Pull request merged**